cosmwasm/packages/enclave-ffi-types/src/types.rs

```diff
@@ -154,6 +154,8 @@ pub enum EnclaveError {
     Panic,
     #[display(fmt = "enclave ran out of heap memory")]
     OutOfMemory,
+    #[display(fmt = "query recursion limit exceeded")]
```

Comment:
This is what is going to be returned to the contract caller, right? Can we make this `depth of nested contract calls exceeded`?

Reply:
yeah, sure. it does sound better :) 
keep in mind though that I think this error will never be displayed.

Reply:
I'm thinking of dev environments where you can make mistakes - I had that happen to me a few times